core-services/prow/02_config/openshift/console-operator/_prowconfig.yaml

```
@@ -88,8 +85,10 @@ tide:
     - master
     labels:
     - approved
-    - bugzilla/valid-bug
+    - docs-approved
```

I recommend that we also add the requirement to link a jira ticket here:

```suggestion
    - docs-approved
    - jira/valid-reference
```

This works fine with all CONSOLE and ODC stories and makes it easier to find the reason for a change later.